src/languageserverinstance.jl

```diff
@@ -15,7 +15,7 @@ type LanguageServerInstance
     function LanguageServerInstance(pipe_in, pipe_out, debug_mode::Bool, user_pkg_dir::AbstractString = haskey(ENV, "JULIA_PKGDIR") ? ENV["JULIA_PKGDIR"] : joinpath(homedir(), ".julia"))
         cache = Dict()
 
-        new(pipe_in, pipe_out, "", Dict{String, Document}(), cache, Channel{Symbol}(500), debug_mode, false, user_pkg_dir)
+        new(pipe_in, pipe_out, "", Dict{String, Document}(), cache, Channel{Symbol}(500), true, false, user_pkg_dir)
```

Comment:
This does not look intentional.
The `debug_mode` parameter is still there, but never used?